source/boundary_temperature/dynamic_core.cc

```diff
@@ -693,30 +693,12 @@ namespace aspect
                 if (cell->face(f)->boundary_id() == CMB_id)
                   {
                     fe_face_values.reinit (cell, f);
+
+                    // Repopulate MaterialInputs, set use_strain_rates to false
+                    in.reinit(fe_face_values, cell, this->introspection(), this->get_solution(), false);
+
                     fe_face_values[this->introspection().extractors.temperature].get_function_gradients (this->get_solution(),
                         temperature_gradients);
-                    fe_face_values[this->introspection().extractors.temperature].get_function_values (this->get_solution(),
-                        in.temperature);
```

Comment:
I guess all of these fields are unused? Or does the code in `reinit()` already set these variables correctly?

Reply:
`reinit` indeed sets these variables correctly, that's why I'm making this change.